pkgs/top-level/python-packages.nix

```diff
@@ -12900,6 +12898,26 @@ in modules // {
     };
   };
 
+  emoji = buildPythonPackage rec {
+    name = "emoji-${version}";
+    version = "0.3.9";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/e/emoji/${name}.tar.gz";
+      sha256 = "19p5c2nlq0w9972rf9fghyswnijwgig5f8cyzs32kppnmzvzbkxw";
+    };
+
+    buildInputs = with self; [ nose ];
+
+    meta = {
+      description = "Emoji for Python";
+      homepage = https://pypi.python.org/pypi/emoji/;
+      license = licenses.bsd3;
+      maintainers = with maintainers; [ joachifm ];
+      platforms = platforms.all;
```

Comment:
`buildPythonPackage` uses `meta.platforms` from the interpreter, which is `platforms = with stdenv.lib.platforms; linux ++ darwin;`, so there is no need to set `platforms`.